File: back-end/skeleton/dotnet/src/DotnetSkeleton.API/DotnetSkeleton.API/Controllers/CoreModules/PingController.cs
using DotnetSkeleton.SharedKernel.Utils;
using Microsoft.AspNetCore.Authorization;
using Microsoft.AspNetCore.RateLimiting;

namespace DotnetSkeleton.API.Controllers.CoreModules;

/// <summary>
/// Controller for handling ping requests.
/// </summary>
[ApiController]
[Route("/api/v{version:apiVersion}/[controller]")]
[ApiVersion("1")]
public class PingController : ControllerBase
{
    /// <summary>
    /// Retrieves a response indicating that the server is alive.
    /// </summary>
    /// <returns>An <see cref="IActionResult"/> representing the HTTP response.</returns>        
    [HttpGet]
    [AllowAnonymous]
    [ProducesResponseType(StatusCodes.Status200OK)]
    [EnableRateLimiting(Constant.SystemInfo.TokenBucketRateLimit)]
    [ApiExplorerSettings(IgnoreApi = true)]
    public IActionResult Get()
    {
        return Ok("Pong");
    }
}

File: back-end/skeleton/nodejs-express/src/controllers/ping.controller.js
/**
 * Ping the server.
 * 
 * @returns TRetrieves a response indicating that the server is alive. 
 */
async function ping(req, res, next) {
    try {
        return res.status(200).json({ message: 'Pong' });
    }
    catch(err) {
        next(err);
    }
}

export {
    ping
};